ui/app/src/views/dashboard/Panel.tsx

```diff
@@ -38,7 +40,8 @@ function Panel(props: PanelProps) {
         height: '100%',
         display: 'flex',
         flexFlow: 'column nowrap',
-        overflow: 'visible',
+        // Hide overflow if StatChart, otherwise set to visible so tooltip is not hidden
+        overflow: isStatsChart ? 'hidden' : 'visible',
```

Comment:
Looks good, I'll hopefully be able to remove this overflow condition in the future once tooltips use Portals.